internal/service/paymentlinksrv/webhook.go

```diff
@@ -2,33 +2,129 @@ package paymentlinksrv
 
 import (
 	"context"
+	"errors"
+	"strconv"
+	"strings"
+	"time"
+
 	aulogging "github.com/StephanHCB/go-autumn-logging"
 	"github.com/eurofurence/reg-payment-cncrd-adapter/internal/api/v1/cncrdapi"
 	"github.com/eurofurence/reg-payment-cncrd-adapter/internal/repository/concardis"
+	"github.com/eurofurence/reg-payment-cncrd-adapter/internal/repository/paymentservice"
 )
 
 func (i *Impl) HandleWebhook(ctx context.Context, webhook cncrdapi.WebhookEventDto) error {
 	aulogging.Logger.Ctx(ctx).Info().Printf("webhook id=%d invoice.paymentRequestId=%d invoice.referenceId=%s", webhook.Transaction.Id, webhook.Transaction.Invoice.PaymentRequestId, webhook.Transaction.Invoice.ReferenceId)
 
 	paylinkId, err := idValidate(webhook.Transaction.Invoice.PaymentRequestId)
 	if err != nil {
+		aulogging.Logger.Ctx(ctx).Error().Printf("webhook called with invalid reference ID. id=%s", webhook.Transaction.Invoice.PaymentRequestId)
 		return err
 	}
 
 	paylink, err := concardis.Get().QueryPaymentLink(ctx, paylinkId)
 	if err != nil {
+		aulogging.Logger.Ctx(ctx).Error().Printf("can't query payment link from concardis. err=%s", err)
```

Comment:
I believe err cannot be printed as %s. Either change the %s to %v, or (my preferred way), use err.Error() to get the error string.